package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vets-json-schema",
-  "version": "3.89.0",
+  "version": "3.90.0",
```

Comment:
This one probably doesn’t need a version bump, but it won’t hurt anything either.